rllib/connectors/agent/state_buffer.py

```diff
@@ -70,11 +89,18 @@ def transform(self, ac_data: AgentConnectorDataType) -> AgentConnectorDataType:
         return ac_data
 
     def to_state(self):
-        return StateBufferConnector.__name__, None
+        # Note(jungong) : it is ok to use cloudpickle here for stats because:
```

Comment:
The reason to use cloudpickle over pickle is simply that you are pickling a data-structure that contains lambda function, right?

Reply:
I actually think we should always use cloudpickle.
It's not officially guaranteed, but there is a less chance of not being able to restore states saved with a higher version of python if we use cloudpickle.
cloudpickle kinda does it automatically for you, using the right pickle library if the version is low (pickle5 etc).